Kittygram/KittyDetailsViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  KittyDetailsViewModel.swift
+//  Kittygram
+//
+//  Created by Hung Nguyen on 3/16/17.
+//  Copyright © 2017 Sunshinejr. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+struct KittyDetailsViewModel {
+    init(repository: Repository) {
+        language.onNext(repository.language)
+    }
+    let language = BehaviorSubject<String?>(value: "")
```

Comment:
Can we move properties before init? I just want to stick to conventions in the project